.buildkite/nightly-benchmarks/scripts/convert-results-json-to-markdown.py

```diff
@@ -187,7 +222,7 @@ def results_to_json(latency, throughput, serving):
         # The GPUs sometimes come in format of "GPUTYPE\nGPUTYPE\n...",
         # we want to turn it into "8xGPUTYPE"
         df["GPU"] = df["GPU"].apply(
-            lambda x: f"{len(x.split('\n'))}x{x.split('\n')[0]}"
+            lambda x: f"{len(x.split(chg_line_char))}x{x.split(chg_line_char)[0]}"
```

Comment:
Why is this change required?

Reply:
used to face a parsing issue on CPU results, but no issue on the latest code. remove the changes.